docs/content/languages/go.md

```diff
@@ -0,0 +1,477 @@
+---
+title: "Using Golang with Fission"
```

Comment:
It's a nitpick, but the Go people dont want people calling their language "golang".  In most places we should use Go instead of Golang, so "Using Go with Fission", etc.